packages/cloud-cognitive/src/components/CreateFullPage/CreateFullPageStep.js

```diff
@@ -69,7 +69,6 @@ export let CreateFullPageStep = forwardRef(
         stepNumber === stepsContext?.currentStep &&
         previousState?.currentStep !== stepsContext?.currentStep
       ) {
-        stepsContext?.setOnNext(onNext);
         stepsContext?.setOnMount(onMount);
       }
     }, [onMount, onNext, stepsContext, stepNumber, previousState?.currentStep]);
```

Comment:
The above commentary also needs updating>

```suggestion
    }, [onMount, stepsContext, stepNumber, previousState?.currentStep]);
```